app/src/main/java/fr/free/nrw/commons/review/ReviewActivity.java

```diff
@@ -100,10 +114,10 @@ protected void onCreate(Bundle savedInstanceState) {
         d[2].setColorFilter(getApplicationContext().getResources().getColor(R.color.button_blue), PorterDuff.Mode.SRC_IN);
 
         if (savedInstanceState != null && savedInstanceState.getParcelable(SAVED_MEDIA) != null) {
-            updateImage(savedInstanceState.getParcelable(SAVED_MEDIA)); // Use existing media if we have one
```

Comment:
Any way to improve these comments rather than just remove them?

Reply:
Sure, I have added java doc to explain why I changed this part, but my teammates pushed a old version, sorry about that. the new version included java doc and kdoc for all changed part.